applications/asset_tracker_v2/src/cloud/azure_iot_hub_integration.c

```diff
@@ -9,6 +9,13 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_CLOUD_INTEGRATION_LOG_LEVEL);
 
+BUILD_ASSERT(IS_ENABLED(CONFIG_AGPS_SRC_SUPL) == 0,
+	     "The Azure IoT Hub integration only supports A-GPS over SUPL. Enable SUPL by setting "
+	     "CONFIG_AGPS_SRC_SUPL");
+
+BUILD_ASSERT(IS_ENABLED(CONFIG_NRF_CLOUD_PGPS) == 0,
```

Comment:
This should be expressible as well in KConfig, CONFIG_AZURE_IOT_HUB=y depends on CONFIG_NRF_CLOUD_PGPS=n

Reply:
Same goes for this one.